src/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {Button} from '@material-ui/core';
+import Login from './Components/Login';
+
+
+function App(){
```

Comment:
Make this into an arrow function. A functional component notation in react doesn't use a `this`. An arrow function doesn't have a lexical `this`. Read more about what `this` is in js